module/Decision/view/decision/member/index.phtml

```diff
@@ -151,6 +151,10 @@ $meetings = $this->meetings;
                             <div class="big-icon fas fa-calendar-alt"></div>
                             <div class="panel-heading"><?= $this->translate('Option calendar') ?></div>
                         </a>
+                        <a href="/corporateidentity" class="panel panel-image">
+                            <div class="big-icon gi gewis-s"></div>
+                            <div class="panel-heading"><?= $this->translate('Corporate Identity') ?></div>
+                        </a>
```

Comment:
You have placed the "button" in the "Manage"-section of the Members page. This section is only visible to "active" users (i.e. users part of an organ). To my understanding, the button is supposed to be visible to all users of the website. Perhaps it should be under the "Association"-section.